client/landing/stepper/constants.ts

```diff
@@ -13,3 +13,20 @@ export const HOW_TO_MIGRATE_OPTIONS = {
 	DO_IT_FOR_ME: 'difm',
 	DO_IT_MYSELF: 'myself',
 };
+
+/**
+ * All Tracks events related to Stepper.
+ * Prefixed with `STEPPER_TRACKS_EVENT_[scope]_[action]` to avoid conflicts with other Tracks events.
+ * Example: `STEPPER_TRACKS_EVENT_STEP_NAV_SUBMIT` -> scope = `STEP_NAV`, action = `SUBMIT`
+ */
+export const STEPPER_TRACKS_EVENT_STEP_NAV_SUBMIT = 'calypso_signup_actions_submit_step'; // TODO clk can be updated to match the constant name
```

Comment:
> // TODO clk can be updated to match the constant name

way way down the line. for now, rather impossible to match the const name with the value, as we are working over existing events in Tracks. A good naming convention though is crucial